addons/report/static/src/js/report.editor.js

```diff
@@ -3,13 +3,17 @@ odoo.define('report.editor', function (require) {
 
 var utils = require('report.utils');
 var editor = require('web_editor.editor');
+var core = require("web.core");
 var options = require('web_editor.snippets.options');
+var ajax = require("web.ajax");
 
 var web_base_url = $('html').attr('web-base-url');
 var trusted_host = utils.get_host_from_url(web_base_url);
 var trusted_protocol = utils.get_protocol_from_url(web_base_url);
 var trusted_origin = utils.build_origin(trusted_protocol, trusted_host);
 
+ajax.loadXML('/web/static/src/xml/base_common.xml', core.qweb);
```

Comment:
Why do you need to load base_common.xml on the report editor ?

It should would work without that no ?